src/wasm.ts

```diff
@@ -0,0 +1,16 @@
+globalThis.crypto = require('crypto').webcrypto;
```

Comment:
This is the actual fix for the Node issue

We require the `crypto` package and assign it to `globalThis.crypto` for the go_wasm file to pickup

Moved this WASM code to a new file so that we didn't need to repeat this 'polyfill' elsewhere